### PR TITLE
Update modify mode when creating PRs via auto flake bot

### PR DESCRIPTION
**Pull request by keyonghan**
Fixes: https://github.com/flutter/flutter/issues/102258

### CODE REVIEW

**Review by christopherfujino — APPROVED** [review]
LGTM

File: app_dart/lib/src/request_handlers/flaky_handler_utils.dart [hunk 1]
```diff
@@ -38,7 +38,7 @@ const String kCiYamlTargetTagsFramework = 'framework';
 const String kCiYamlTargetTagsHostonly = 'hostonly';
 
 const String kMasterRefs = 'heads/master';
-const String kModifyMode = '100755';
```
Comment (chunhtai) [inline app_dart/lib/src/request_handlers/flaky_handler_utils.dart:41]: what does the new number do? maybe add a comment as well

**Review by chunhtai — APPROVED** [review]
LGTM

File: app_dart/lib/src/request_handlers/flaky_handler_utils.dart (hunk 1 above)
Comment (keyonghan) [reply app_dart/lib/src/request_handlers/flaky_handler_utils.dart:41]: Added a comment to explain.

Context: the bot is using `100755` which updates the .ci.yaml file to be executable. And this causes `analyze` failing with recent PR: https://github.com/flutter/flutter/pull/101853

**Review by jmagman — COMMENTED** [review]
Why is the bot modifying the mode in the first place?

**Comment by keyonghan** [comment]
> Why is the bot modifying the mode in the first place?

/cc @chunhtai may have a better idea.

**Pull request merged**